src/mobile-token/utils.ts

```diff
@@ -48,7 +47,24 @@ export const wipeToken = async (tokensIds: string[], traceId: string) => {
        * already removed, in this case, just wipe the local token.
        */
       if (isEntityDeletedError(err)) {
-        await mobileTokenClient.clear();
+        notifyBugsnag(Error(`Token removed from backoffice ${id}`), {
+          errorGroupHash: 'token',
+          metadata: {
+            traceId,
+            description: `Token already deleted in backoffice ${id}, start new`,
+          },
+        });
+        break; // No need to do anything, proceed to clear the local token
```

Comment:
This part here with the `break` isn't really intuitive. Why don't we need to iterate over the other token ids if one of them is already deleted? 🤔

Reply:
Right, this one should be left at do nothing and not `break`, to allow continuing removal of other tokens.